scala-3-project-template/README.md

```
@@ -0,0 +1,8 @@
+## sbt project compiled with Scala 3
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)